### DISCUSSION TITLE
Rules: Energy#Power doesn't work, but Energy#Current does

### DISCUSSION THREAD

**Discussion by GijsTimmers** [arendst/Tasmota#15801] [Q&A] [answered]
Running Tasmota 11.0.0 on an ESP8266 device

Trying to send a message when a device gets powered down, measured with a POW.

The following piece of code works:
```
Rule1 5
Rule1
  ON Energy#Current<0.067 DO WebQuery <url> POST <device off> ENDON
  ON Energy#Current>0.067 DO WebQuery <url> POST <device on> ENDON
```
The following piece of code doesn't:

```
Rule1 5
Rule1
  ON Energy#Power<6 DO WebQuery <url> POST <device off> ENDON
  ON Energy#Power>6 DO WebQuery <url> POST <device on> ENDON
```

The Current and Power values correspond. I feel like I made a syntax error. What could it be?

**Comment by barbudor** [comment, marked as answer]
I don't see any obvious syntax error. It should work unless you have typed in some invisible character that breaks it.
I would invite you to carefully retype the rule (and not copy/paste it from any text file you saved it to)

Also I would imagine you meant 'Rule1 5' to enable the 'once' mode

**Reply by GijsTimmers** [reply]
Thank you, I updated it

**Reply by sfromis** [reply]
A couple of things to check... Compare with the JSON format you get in the console. The rule triggers should exactly match the object structure. Of course, also if the rules are actually activated in both situations. And watch the console with `Weblog 3` for details of what Tasmota actually acts on.

More generally, I'd recommend at least a bit of hysteresis instead of having the on/off triggers compare to the same number.
